### DISCUSSION TITLE
Prior monte Carlo and ensemble size

### DISCUSSION THREAD

**Discussion by d-strom** [gmdsi/GMDSI_notebooks#122] [Q&A] [answered]
Hi,

I watched your excellent videos in the New Zealand session and was caught up in the discussion about prior Monte Carlo (during part 3).

Jeremy describes prior data conflict and how this can be evaluated during the prior Monte Carlo step. And further, how one can interpret the results.

If our ensemble does not cover the observations, then we have a prior data conflict and have reason to reconsider our conceptual model, boundary conditions, etc.
But how do we know if our, say, 250 ensemble draws cover the value we are trying to simulate?
How is the size of the ensemble for prior Monte Carlo determined?
If, for example, we have a model with pilot points, several layers, several types of parameters, and thus have 3500 parameters. Is 250 ensemble members enough?
My immediate feeling is that it does not introduce sufficient variation to ensure that all parts of the model are represented in a sufficiently broad range to know that it covers my 1-1 line. Or am I misunderstanding the concept of prior Monte Carlo?

sincerely 
David

ps. I apologieze of I have been bombing you with questions lately. The discussions have been very helpful!

**Comment by jtwhite79** [comment, marked as answer]
I think all this prior-data conflict stuff (like just about everything else related to applied groundwater modeling) should be taken as approximate and as an indicator only.    Of course you can always run more realizations and possibly stumble onto a realization that gives some outlier-type behavior that is more similar to the observed values. But I think if you've run fews 10s to 100s of realizations and those results arent very close to the observed values, that's telling you something.  

That being said, if you really want to get technical with things, you could do a convergence analysis with the ensemble were you calculate a mean, stdev, 95% CL etc for 10 randomly selected output/obs realizations, then repeat these summary stats as you select increasingly more random realizations (ie. 20, 50, 100, etc).  Ive done this a few times and usually the mean and stdev stabilize relatively quickly, but those extreme values/events almost never stabilize...hence direct predictive hypothesis testing...
